### DISCUSSION TITLE
Inspiration

### DISCUSSION THREAD

**Discussion by trusktr** [felixmariotto/three-mesh-ui#135] [Ideas]
I thought to share this neat project: http://makepad.nl/

It is written in Rust, and compiled to WebAssembly. But the neat part is it ultimately renders all the UI (tree view, scrollable views, editable text, etc) with WebGL.

**Comment by timothyallan** [comment]
The scrolling, text selection features and menu animations are all amazing. 
![Screen Shot 2020-08-20 at 9 02 20 PM](https://user-images.githubusercontent.com/1995135/90847843-b26e6e00-e328-11ea-8d31-b1190bdca6de.png)
I was super impressed with their clever dynamic `textarea` input trick.

**Reply by sylwesterdigital** [reply]
521: Web server is down
![Screenshot 2022-01-21 at 12 37 34](https://user-images.githubusercontent.com/27820237/150528309-2cd066e6-dde9-4802-8afa-54d0f2609d4e.png)
